tests/unit/AnnualReport.spec.ts

```diff
@@ -2077,9 +2077,9 @@ describe('Annual Report - payment required error', () => {
 
   beforeEach(() => {
     // init store
-    store.state.identifier = 'CP0001191'
-    store.state.entityType = 'CP'
-    store.state.entityName = 'Legal Name - CP0001191'
+    store.state.business.identifier = 'CP0001191'
+    store.state.business.legalType = 'CP'
+    store.state.business.legalName = 'Legal Name - CP0001191'
```

Comment:
When we're writing new tests, what do you think about setting the state by calling a mutator rather than mutating the state directly.  Calling a mutator gives us flexibility to change our store in the future without having to refactor larger number of tests.

Reply:
I'll update the unit tests I touched in my next PR.

Reply:
I'd call a mutation directly using something like this `store.commit('mutationName', newValue)`.  There shouldn't be any need to `await` since mutations are synchronous.  Depending on what store object you need to set, it might be handy to create a mutation specifically for testing.

Reply:
^^ That's what I'm finding: I need to create some business property mutations so that I don't set the entire business object every time.

Reply:
I updated the unit tests in my next PR: https://github.com/bcgov/business-filings-ui/pull/461